content/platforms/release-notes/k8s-6-0-8-20-2020-12.md

```diff
@@ -0,0 +1,93 @@
+
+## Images
+
+ * **Redis Enterprise** - redislabs/redis:6.0.8-30 or redislabs/redis:6.0.8-30.rhel7-openshift
+ * **Operator** - redislabs/operator:6.0.8-20
+ * **Services Rigger** - redislabs/k8s-controller:6.0.8-20 or redislabs/services-manager:6.0.8-20 (Red Hat registry)
+
+## New features
+
+ * Databases custom resources (REDB) can be created in separate consumer namespaces from the operator and cluster namespace. The operator deployment is configured to watch specific namespaces for these database REDB CR’s.
+ * The [Gesher admission control proxy](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/admission/GESHER.md) is now certified by Red Hat.
+ * REDB CR’s no longer require a Redis Enterprise cluster name. The name will default to the cluster in the context of the operator.
+ * REC and REDB CR’s are now validated via a schema.
+
+## Important fixes
+
+ * Using database controller (REDB) no longer generates errors:  “failed to update database status" (RED44919)
+ * Issues with configuring replica-of through the database controller (REDB) and TLS have been fixed. (RED48285)
+ * A time out issue with “rlutil upgrade” was fixed. (RED48700)
+
+## Known limitations
+
+### CrashLoopBackOff causes cluster recovery to be incomplete  (RED33713)
+
+When a pod status is CrashLoopBackOff and we run the cluster recovery, the process will not complete. The workaround is to delete the crashing pods manually and recovery process will continue.
+
+### Long cluster names cause routes to be rejected  (RED25871)
+
+A cluster name longer than 20 characters will result in a rejected route configuration as the host part of the domain name exceeds 63 characters. The workaround is to limit cluster name to 20 characters or less.
+
+### Cluster CR (REC) errors are not reported after invalid updates (RED25542)
+
+A cluster CR specification error is not reported if two or more invalid CR resources are updated in sequence.
+
+### An unreachable cluster has status running (RED32805)
+
+When a cluster is in an unreachable state the state is still running instead of being reported as an error.
+
+### Readiness probe incorrect on failures (RED39300)
+
+STS Readiness probe doesn't mark a node as not ready when rladmin status on the node fails.
+
+### Role missing on replica sets (RED39002)
+
+The redis-enterprise-operator role is missing permission on replica sets.
+
+### Private registries are not supported on OpenShift 3.11 (RED38579)
+
+Openshift 3.11 doesn't support dockerhub private registry. This is a known OpenShift issue.
+
+### Internal DNS and Kubernetes DNS may have conflicts (RED37462)
+
+DNS conflicts are possible between the cluster mdns_server and the K8s DNS. This only impacts DNS resolution from within cluster nodes for Kubernetes DNS names.
```

Comment:
@alexmilowski Can you elaborate on what is NOT impacted by this conflict?

Reply:
This has been in every release note that has been published for quite awhile. People understand what this means.